packages/shared/src/hooks/squads/useSquadActions.ts

```diff
@@ -0,0 +1,17 @@
+import { useMemo } from 'react';
+import { useMutation, useQueryClient } from 'react-query';
+import { updateSquadMemberRole } from '../../graphql/squads';
+
+interface UseSquadActions {
+  onUpdateRole?: typeof updateSquadMemberRole;
+}
+
+export const useSquadActions = (): UseSquadActions => {
+  const client = useQueryClient();
+  const { mutateAsync: onUpdateRole } = useMutation(updateSquadMemberRole, {
```

Comment:
Should it not just be `updateRole` as it actually executes and not just "After" handles it?

Reply:
Our exposed functions have a prefix `on`, just keeping it in line with the code base. I think what you mean is about present tense vs past tense for which we use present tense as it is appropriate as you explained.